pyproject.toml

```diff
@@ -10,7 +10,19 @@ package-mode = false
 python = "^3.10"
 celery = {extras = ["redis"], version = "^5.4.0"}
 openrelik-worker-common = "^2024.11.27"
+pandas = "^2.2.3"
+pyparsing = "^3.2.0"
+
+[tool.poetry.group.dev.dependencies]  
+pytest = "^7.4"  
+pytest-sugar = "*"  
+pytest-cov = "*"  
 
 [build-system]
 requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.pytest.ini_options]
```

Comment:
I don't think this is needed. pytest will pickup any file that startes with test_* autmatically.